nipype/algorithms/tests/test_confounds.py

```diff
@@ -1,12 +1,20 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-
-from nipype.testing import (assert_equal, example_data)
-from nipype.algorithms.confounds import FramewiseDisplacement
+import os
+from nipype.testing import (assert_equal, example_data, skipif)
+from nipype.algorithms.confounds import FramewiseDisplacement, ComputeDVARS
 import numpy as np
 from tempfile import mkdtemp
 from shutil import rmtree
 
+nonitime = True
+try:
+    import nitime
+    nonitime = False
```

Comment:
We generally use this convention for boolean variables that will be used in skip decorators.